mockingServer/modules/RequestHandler.py

```diff
@@ -0,0 +1,14 @@
+from flask.views import MethodView
+from flask import request
+import json
+
+
+class RequestHandler(MethodView):
+    def __init__(self):
+        with open('mockingServer/data/login.json', 'r') as data:
+            self.session_id = json.load(data)
```

Comment:
Performance is not really an issue here, but it does open this file every time as we are issuing requests, which is quite often. I would be finde with issuing a static value from here, we are having it in a file, which is not changing anyway, so it has no real benefit.

Reply:
True, I will hardcode it and remove the login.json from the data subdirectory.

Reply:
done